ui/raidboss/data/02-arr/raid/t1.ts

```diff
@@ -7,11 +7,16 @@ import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 
 export interface Data extends RaidbossData {
-  started?: boolean;
+  started: boolean;
 }
 
 const triggerSet: TriggerSet<Data> = {
   zoneId: ZoneId.TheBindingCoilOfBahamutTurn1,
+  initData: () => {
```

Comment:
Hardly a nit, since I like the current pattern, but before this rolls out to everywhere does anyone prefer:
```js
initData: () => ({
  started: false,
}),
```

Reply:
Both work, so it's more a matter of style preference for the reference trigger?